subset/bacnet/bacnetTests/src/main/java/FauxDeviceEngine/EntryPoint.java

```diff
@@ -0,0 +1,129 @@
+package FauxDeviceEngine;
+
+import com.serotonin.bacnet4j.LocalDevice;
+import com.serotonin.bacnet4j.exception.BACnetServiceException;
+import com.serotonin.bacnet4j.npdu.ip.IpNetwork;
+import com.serotonin.bacnet4j.obj.BACnetObject;
+import com.serotonin.bacnet4j.transport.Transport;
+import com.serotonin.bacnet4j.type.enumerated.ObjectType;
+import com.serotonin.bacnet4j.type.enumerated.PropertyIdentifier;
+import com.serotonin.bacnet4j.type.primitive.CharacterString;
+import helper.FileManager;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+public class EntryPoint {
+
+    private static int deviceId = 0;
+    private static IpNetwork network;
+    private static LocalDevice localDevice;
+    private static String fauxDeviceJSONFilename = "";
+    private static int timeout = 1000;
+
+    public static void main(String[] args) {
+        if (args.length != 3) {
+            throw new RuntimeException("Usage: localIpAddr broadcastIpAddr fauxDeviceJSONFilename");
+        }
+        String localIpAddr = args[0];
+        String broadcastIpAddr = args[1];
+        fauxDeviceJSONFilename = args[2];
+
+        int port = IpNetwork.DEFAULT_PORT;
```

Comment:
Not to nitpick, but why assign it to a separate var instead of using it directly.

Reply:
we have to name everything verbosely so someone can follow what the code intentions are

Reply:
`IpNetwork.DEFAULT_PORT` is already quite verbose